discord/message.py

```diff
@@ -998,7 +998,7 @@ def is_system(self) -> bool:
         )
 
     @utils.cached_slot_property('_cs_system_content')
-    def system_content(self):
+    def system_content(self) -> str:
```

Comment:
There has to be a better way to do this than chained ifs no?

Reply:
Match-case crossed my mind but that's not an option for now, at least.